pyanaconda/ui/gui/spokes/user.py

```diff
@@ -118,8 +117,8 @@ def refresh(self):
         self._tHome.set_text(homedir)
         self._origHome = homedir
 
-        self._cUid.set_active(self.user.uid != USER_UID_NOT_SET)
-        self._cGid.set_active(self.user.gid != USER_GID_NOT_SET)
+        self._cUid.set_active(self.user.uid_status != constants.ID_NOT_SET)
+        self._cGid.set_active(self.user.gid_status != constants.ID_NOT_SET)
```

Comment:
The same here, just create a method in the structure class for this check.

Reply:
I don't think we need an extra method just for the check. We might be able to use `get_uid` and `get_gid` as suggested above.